web_widget_darkroom/static/lib/darkroomjs/core/darkroom.js

```diff
@@ -0,0 +1,356 @@
+/**
+*    Copyright 2013 Matthieu Moquet
+*    Copyright 2016-2017 LasLabs Inc.
+*    License MIT (https://opensource.org/licenses/MIT)
+**/
```

Comment:
Yeah it's still best to have it here - it's relevant in the context of the code to know what version you're dealing with.

Also hmmm I wonder if we should ignore `static/lib` from our linter. We don't really want to have to modify libraries like this.